src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/AggregationDeserializer.cs

```diff
@@ -1,35 +1,20 @@
 ﻿using System;
 using Microsoft.Extensions.Logging;
 using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
-using YamlDotNet.RepresentationModel;
 
 namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Core
 {
     public class AggregationDeserializer : Deserializer<AggregationV1>
     {
         private const string IntervalTag = "interval";
 
-        private readonly TimeSpan _defaultAggregationInterval = TimeSpan.FromMinutes(5);
+        private static readonly TimeSpan DefaultAggregationInterval = TimeSpan.FromMinutes(5);
 
         public AggregationDeserializer(ILogger<AggregationDeserializer> logger) : base(logger)
         {
+            MapOptional(aggregation => aggregation.Interval, DefaultAggregationInterval);
```

Comment:
I like this a lot, thanks!

Reply:
Thanks - it definitely simplifies writing the deserializers. I was going for something similar to the way that you configure AutoMapper or FluentValidation, but I didn't want to go crazy and take things too far, which is why I've taken the approach of just using optional parameters for setting the default, etc instead of doing something like `Map(aggregation => aggregation.Interval).WithDefault(...).Required()`.